drivers/homeless_summary_report/app/models/homeless_summary_report/report.rb

```diff
@@ -110,8 +110,18 @@ def multiple_project_types?
       true
     end
 
-    def default_project_types
-      [:ph, :es, :th, :sh, :so]
+    def project_type_ids
+      [:es, :so, :sh, :th, :ph].map { |s| GrdaWarehouse::Hud::Project::PERFORMANCE_REPORTING[s.to_sym] }.flatten
```

Comment:
Maybe pull the array out? I'm guessing this list and the one in `project_type_options_for_select` should match?

Reply:
Is this the right way to do that in Ruby? https://github.com/greenriver/hmis-warehouse/pull/2199/commits/2c4fe4fce6534531a76504db5b2db129d3e55a25

Reply:
Looks good to me!